### ISSUE TITLE
[BUG] Error while using model.simulate with unknown control input function

### ISSUE THREAD

**Issue by pingkung** [dynamicslab/pysindy#100]
Hi there,

I downloaded the example code from the pysindy website. In the example file named "1_feature_overview.py", there is an error while I tried to run it in the function of simulate (line 568) as mentioned below.
### Code from example:

```python
x_test_sim = model.simulate(x0_test, t_test, u=u_test)
```

### Error message:

ValueError: A value in x_new is above the interpolation range.
### PySINDy/Python version information:
1.2.2

### How can I fixed it:
From the "pysindy.py" at line 725, I added the further parameter to the function "interp1d" as suggested from https://stackoverflow.com/questions/45429831/valueerror-a-value-in-x-new-is-above-the-interpolation-range-what-other-re?answertab=votes#tab-top

Previous code
```python
u_fun = interp1d(t, u, axis=0, kind="cubic")
```
Modified code
```python
u_fun = interp1d(t, u, axis=0, kind="cubic", fill_value="extrapolate")
```

After I put it, the simulate function is work for me. Maybe you consider to put it into the update version of pysindy.

Regards,

**Comment by briandesilva** [comment]
Thanks for bringing this up. I'll investigate and put out a fix in a few hours.

**Comment by briandesilva** [comment]
Fixed in 7afc3bcaea9363a48276ad12c4a7dd9ff02feef4.

I've created a new release (v1.3.0) which incorporates the fix, so you should be able to get it with `pip install --upgrade pysindy` once the changes propagate to PyPI in a few minutes.
